.github/workflows/ci_notebooks.yml

```diff
@@ -0,0 +1,48 @@
+name: Run pytest on Notebooks
+
+on:
+  workflow_dispatch:
+
+  pull_request:
+    paths:
+      - '**/*.ipynb'  # Adjust the path pattern to match your project's Python files
```

Comment:
Is this the path we want?

Reply:
this is for testing `any` notebook that is pushed to the repo (not only on the specific path).

Breaking it down, `**/` means any path where `*.ipynb` whatever is a notebook